src/test/java/nextstep/subway/fixture/acceptance/when/StationApiFixture.java

```diff
@@ -22,6 +22,19 @@ public abstract class StationApiFixture {
                 .extract();
     }
 
+    public static long 지하철역_생성_요청_후_id_추출(String name) {
+
+        Map<String, String> params = 지하철역_등록_요청_데이터_생성(name);
+
+        return RestAssured.given().log().all()
+            .body(params)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when().post("/stations")
+            .then().log().all()
+            .extract()
+            .jsonPath().getLong("id");
+    }
```

Comment:
지하철역 생성하는 메서드는 이미 존재하니 재활용해도 좋을 것 같아요 
```suggestion
    public static long 지하철역_생성_요청_후_id_추출(String name) {
        return 지하철역_생성_요청(name).jsonPath().getLong("id");
    }
```

Reply:
오..! 이렇게 유연한 방법을 생각치도 못했네요 정말 감사합닏다..😂